tests/package.json

```
@@ -0,0 +1,14 @@
+{
+    "scripts": {
+        "test:e2e": "cucumber-js --require cucumber.conf.js --require e2e/stepDefinitions/**/*.js --format @cucumber/pretty-formatter"
```

should these setup files also go to `tests/e2e` folder? just my thought
```
tests/
  - e2e/
    - cucumber.conf.js
    - package.json
    - package-lock.json
```
```suggestion
        "test:e2e": "cucumber-js --require cucumber.conf.js --require stepDefinitions/**/*.js --format @cucumber/pretty-formatter"
```

---

yeah, that also a good idea